packages/design-system-docs/src/pages/components/Autocomplete/Autocomplete.example.jsx

```diff
@@ -2,6 +2,17 @@ import { Autocomplete, TextField } from '@design-system';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
+// Right now the documentation files don't actually get parsed for styles, so
+// this is a hack to add custom CSS classes for this example.
+const styleElement = document.createElement('style');
+styleElement.innerText = `
+  .ds-c-autocomplete__search-all.ds-c-autocomplete__list-item--active a {
+    color: #fff;
+    text-decoration-color: #fff;
+  }
+`;
+document.querySelector('head').appendChild(styleElement);
```

Comment:
This is pretty hacky, and I'm not proud of it. Anyone have ideas to work around this limitation for now? Or ideas for any other kind of alternative? (I realize it's also not necessary for the example to even have the `ds-c-autocomplete__search-all` class because I'm applying the style to the link element, but it's showcasing the custom classes.

Reply:
Could you put a css class on the `<div>` (line 17) like `example-autocomplete`
And then in the file `packages/design-system-docs/src/styles/components/_example.scss`, you could move the styles to be
```
.example-autocomplete{
  .ds-c-autocomplete__search-all.ds-c-autocomplete__list-item--active a {
      color: #fff;
      text-decoration-color: #fff;
    }
}
```

Reply:
That's great that there's a canonical place for these! So much better